### PR TITLE
Store changes to DB

### PR DESCRIPTION
**Pull request by joybro**
This will be used to show page change history.

### CODE REVIEW

File: server/db/dynamoDB.js [hunk 1]
```diff
@@ -108,9 +110,42 @@ const readUser = async (doc: ReadUser): Promise<User> => {
   };
 
   try {
-    const result: UserQueryResult = await docClient().query(params).promise();
+    const result: QueryResult = await docClient().query(params).promise();
```
Comment (bongsok) [inline server/db/dynamoDB.js]: Why are the types of Article and User query result merged? It looks object types in array are different.

File: server/server.jsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ import cookieParser from 'cookie-parser';
 import createReduxStore from '../modules/store';
 import db from './db/db';
 import expressJwt, { UnauthorizedError as Jwt401Error } from 'express-jwt';
+import jsdiff from 'diff';
```
Comment (bongsok) [inline server/server.jsx]: Here jsdiff seems not to be used anywhere.

File: server/db/dynamoDB.js (hunk 1 above)
Comment (joybro) [reply server/db/dynamoDB.js]: Thanks. I missed that part.

File: server/server.jsx (hunk 2 above)
Comment (joybro) [reply server/server.jsx]: You're right. I updated both of them in the new revision.

**Pull request merged**